src/printer.ts

```diff
@@ -393,6 +394,7 @@ export class PugPrinter {
 	private formatText(text: string): string {
 		let result: string = '';
 		while (text) {
+			// Find double curly brackets
```

Comment:
Refactor this entire method into smaller parts (that can be reused and are easier to grasp)

Reply:
Okay, this code hurts, but I personally decided for now to await feedback from svelte community if there are further errors and printed warnings in their console output
Based on this feedback I can refactor this messy code
But now it's working and the svelte community can start working our formatter